horace_core/sqw/PixelData/@PixelData/private/logical_to_normal_index_.m

```diff
@@ -0,0 +1,15 @@
+function abs_pix_indices = logical_to_normal_index_(obj, logical_indices)
+%ABS_PIX_INDICES Convert the given logical indices to normal indices
```

Comment:
What does this mean? Example?

Reply:
Is it converting:
`[true, true, false, false, true]`
to
`[1,3,5]`?

Reply:
I've added an example to the function help - hopefully that clarifies things